src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import core.basesyntax.Ball.Color;
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
```

Comment:
move random to the class field, like
![image](https://user-images.githubusercontent.com/12864128/210410246-5a4fa6bb-eb1a-4c08-9312-c21aa8168d29.png)

Reply:
done